client/rest-high-level/src/main/java/org/elasticsearch/client/MachineLearningClient.java

```diff
@@ -166,4 +168,42 @@ public void openJobAsync(OpenJobRequest request, RequestOptions options, ActionL
             listener,
             Collections.emptySet());
     }
+
+    /**
+     * Closes Machine Learning Jobs
+     * Closes one or more jobs. A job can be opened and closed multiple times throughout its lifecycle.
+     *
+     * A closed job cannot receive data or perform analysis operations, but you can still explore and navigate results.
+     *
+     * @param request request containing jobIds and additional optional options
```

Comment:
I don't think we should use `jobIds` here. We refer to it as `job_id` in the open job doc. Perhaps rephrase like `request containing the `job_id` for each job to close and additional optional options`. Hm, typing `optional options` also felt funny. Perhaps remove `optional` from both?